### DISCUSSION TITLE
Customize OSDP readers LED behavior, A1001

### DISCUSSION THREAD

**Discussion by DanAloud** [AxisCommunications/discussions#189] [VAPIX]
Hello!

 We are trying to figure out a way to customize the LED behavior for OSDP readers connected to a A1001. We have developed a integration that uses FeedbackMessage and can pick the profile we want, however we need to edit the profile it self. We have some readers that will be mounted outside that will be visible for the public, and do not want the reader to light bright green when door is opened. So we would like to be able to edit one profile (or prefarably all of them for further customization) and turn off all LED:s. Is it possible somehow?

**Comment by vivekatoffice** [comment]
Customizing the LED behavior on OSDP readers connected to the A1001 often requires directly modifying the reader’s configuration settings rather than controlling these through standard feedback profiles, as the A1001 typically uses pre-defined behavior templates.

More details about [FeedbackMessage command](https://www.axis.com/vapix-library/subjects/t10067552/section/t10067528/display?section=t10067528-t10067902) are available in VAPIX APIs.

Product discontinuation: [AXIS A1001 Network Door Controller](https://www.axis.com/products/axis-a1001/support)
